src/sentry/static/sentry/app/utils/consolidatedScopes.jsx

```diff
@@ -0,0 +1,137 @@
+import {pull, groupBy} from 'lodash';
+
+const PERMISSION_LEVELS = {
+  read: 0,
+  write: 1,
+  admin: 2,
+};
+
+const HUMAN_RESOURCE_NAMES = {
+  project: 'Project',
+  team: 'Team',
+  release: 'Release',
+  event: 'Event',
+  org: 'Organization',
+  member: 'Member',
+};
+
+const PROJECT_RELEASES = 'project:releases';
+
+export default class ConsolidatedScopes {
```

Comment:
Does this need to be a class? We don't seem to keep/re-use any state within the class. 🤷‍♂️

Reply:
I guess not. We could just expose the two interface functions instead.